### ISSUE TITLE
Crash on entering server or infinite loading on 83%/PC freeze

### ISSUE THREAD

**Issue by alex-schlecht** [GTNewHorizons/GT-New-Horizons-Modpack#14665]
### Your GTNH Discord Username

coldnesspwn

### Your Pack Version

2.4.0

### Your Server

private server

### Java Version

Java 8

### Type of Server

None

### Your Actions

Trying to load game or enter server
When loading freeze at 83%, it says starting sound engine
[hs_err_pid3100.log](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/12836722/hs_err_pid3100.log)

### Crash Report

[fml-client-latest.log](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/12836725/fml-client-latest.log)

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the crash still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this crash consistently by following the exact steps I described above.
- [X] I have asked other people and they confirm they also crash by following the exact steps I described above.

**Comment by alex-schlecht** [comment]
OS win11, 
![ApplicationFrameHost_NmoCdm5IqK](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/113642938/3b7925bf-31a4-46ef-89a2-377bf8fb3d86)

**Comment by alex-schlecht** [comment]
How "freeze loading" part looks like in console
`[11:45:48] [Client thread/INFO] [STDOUT]: [pcl.opensecurity.client.sounds.AlarmResource:generateSoundsJSON:81]: {"StargateAtlantisTheme":{"category":"master","sounds":[{"name":"\\mods\\OpenSecurity\\sounds\\alarms\\StargateAtlantisTheme","stream":false}]},"klaxon2":{"category":"master","sounds":[{"name":"\\mods\\OpenSecurity\\sounds\\alarms\\klaxon2","stream":false}]},"alarm-SGA":{"category":"master","sounds":[{"name":"\\mods\\OpenSecurity\\sounds\\alarms\\alarm-SGA","stream":false}]},"alarm-SG1evac":{"category":"master","sounds":[{"name":"\\mods\\OpenSecurity\\sounds\\alarms\\alarm-SG1evac","stream":false}]},"klaxon1":{"category":"master","sounds":[{"name":"\\mods\\OpenSecurity\\sounds\\alarms\\klaxon1","stream":false}]},"alarm-SGAoffworld":{"category":"master","sounds":[{"name":"\\mods\\OpenSecurity\\sounds\\alarms\\alarm-SGAoffworld","stream":false}]},"alarm-SG1":{"category":"master","sounds":[{"name":"\\mods\\OpenSecurity\\sounds\\alarms\\alarm-SG1","stream":false}]}}
[11:45:48] [Client thread/INFO] [betterloadingscreen]: BLS Splash loading thread closing
java.lang.Throwable
	at alexiil.mods.load.MinecraftDisplayer.close(MinecraftDisplayer.java:1509) [MinecraftDisplayer.class:?]
	at alexiil.mods.load.MinecraftDisplayerWrapper.close(MinecraftDisplayerWrapper.java:35) [MinecraftDisplayerWrapper.class:?]
	at alexiil.mods.load.ProgressDisplayer.close(ProgressDisplayer.java:225) [ProgressDisplayer.class:?]
	at cpw.mods.fml.client.SplashProgress.finish(SplashProgress.java) [SplashProgress.class:?]
	at net.minecraft.client.Minecraft.displayInitErrorScreen(Minecraft.java:5248) [bao.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:5099) [bao.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:148) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_381]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_381]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_381]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_381]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_381]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_381]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_381]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_381]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:243) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:278) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
[11:45:48] [Client thread/INFO] [betterloadingscreen]: Turned Forge splash screen on in splash.properties
[11:46:27] [Sound Library Loader/INFO] [STDOUT]: [paulscode.sound.SoundSystemLogger:importantMessage:90]:     ERROR MESSAGE:
[11:46:27] [Sound Library Loader/INFO] [STDOUT]: [paulscode.sound.SoundSystemLogger:message:69]:         SoundSystem did not load after 30 seconds.
[11:46:27] [Sound Library Loader/INFO] [STDOUT]: [paulscode.sound.SoundSystemLogger:message:69]: 
[11:46:27] [Sound Library Loader/INFO] [STDOUT]: [paulscode.sound.SoundSystemLogger:message:69]: Starting up SoundSystem...
[11:46:28] [Thread-27/INFO] [STDOUT]: [paulscode.sound.SoundSystemLogger:message:69]: Switching to No Sound
[11:46:28] [Thread-27/INFO] [STDOUT]: [paulscode.sound.SoundSystemLogger:message:69]:     (Silent Mode)
[11:46:28] [Sound Library Loader/INFO] [STDOUT]: [paulscode.sound.SoundSystemLogger:message:69]: 
[11:46:28] [Sound Library Loader/INFO]: Sound engine started`

**Comment by chochem** [comment]
there is a problem with your jvm.dll. So you probably want to reinstall java or try a different one.

**Comment by alex-schlecht** [comment]
> there is a problem with your jvm.dll. So you probably want to reinstall java or try a different one. (of course it could also be a RAM issue, but lets hope not)

well, ive tryed multiple java versions

**Comment by ilia-21** [comment]
I have the same problem. Tried several versions of java, same result. Yesterday it was loading fine, I'll try reinstalling and update the comment if that fixes it

**Comment by Kiefy** [comment]
I'm having this exact issue now in 2024.  Like others, tried multiple JVM's from 8 to 22.

Sound worked the first time I ran the game, then never again.

**Comment by boubou19** [comment]
check your audio card drivers, sometimes it's just this.

**Comment by Kiefy** [comment]
I notice the sound works in vanilla, just not GTNH.

**Comment by Kiefy** [comment]
I think I discovered that this freeze is an odd side effect of disabling Ender Zoo.

**Comment by Roaring1** [comment]
I had this before. I just looked further into it.

I fixed it by one of three things.

I have a Nvidia Graphics Card (3070) . I updated their drivers (via the Nvidia App).
I use Avast Driver Updater (premium). I updated the Realtek audio driver with this.
then, I restarted my PC.
After restarting, I started MMC, and Gretech. It worked.

**So, if you have this issue, try to update your GPU driver, or your Audio Driver. Or, just even try restarting your computer..**


Side note: I used g2a for an Avast premium driver updater key; a year for like $5. very worth it. same with Cleanup Premium.
